packages/core/helper-plugin/src/icons/SortIcon.tsx

```diff
@@ -0,0 +1,20 @@
+import * as React from 'react';
+
+import { CarretDown } from '@strapi/icons';
+import styled from 'styled-components';
+
+export interface SortIconProps extends React.SVGProps<SVGSVGElement> {
+  isUp: boolean;
+}
+
+const transientProps: Partial<Record<keyof SortIconProps, boolean>> = {
```

Comment:
Couldn't we just let TypeScript infer this type? It feels a bit verbose for the one key we are using but maybe there is something I'm missing.

Reply:
hmm...we are referring it as `transientProps[prop]` , and typescript suggests `(parameter) prop: keyof SortIconProps`. So we need to define it explicitly as above.

Reply:
ok works for me :)